kahypar/partition/partitioner.h

```diff
@@ -90,6 +90,7 @@ static inline void partition(Hypergraph& hypergraph, const Context& context) {
 class Partitioner {
  private:
   static constexpr bool debug = false;
+  static constexpr HypernodeID LARGE_HE_THRESHOLD = 1000;
```

Comment:
Would it make sense to expose this as a config parameter?

Reply:
Could you still add it as a tuning parameter please? That way, it at least becomes visible from the outside and doesn't get lost in somewhere in the code.

Reply:
done